includes/Modules/Analytics/Settings.php

```diff
@@ -121,11 +121,24 @@ function ( $option ) {
 				 * @param bool $adsense_linked Null by default, will fallback to the option value if not set.
 				 */
 				$adsense_linked = apply_filters( 'googlesitekit_analytics_adsense_linked', null );
-
 				if ( is_bool( $adsense_linked ) ) {
 					$option['adsenseLinked'] = $adsense_linked;
 				}
 
+				/**
+				 * Filters the state of the can use snipped setting.
+				 *
+				 * This filter exists so that useSnippet can be restored to true when the Tag Manager module
+				 * is disconnected, ensuring the Analytics snippet is always included.
+				 *
+				 * @since n.e.x.t
+				 * @param bool $can_use_snippet Null by default, will fallback to the option value if not set.
```

Comment:
This description is slightly inaccurate now.
```suggestion
				 * @param bool $can_use_snippet Whether or not `useSnippet` can control snippet output. Default: `true`.
```